client/jetpack-cloud/sections/pricing/jpcom-footer/index.tsx

```diff
@@ -234,7 +234,7 @@ const JetpackComFooter: React.FC = () => {
 					},
 					{
 						label: translate( 'Press' ),
-						href: addQueryArgs( utmParams, 'https://automattic.com/press/' ),
+						href: 'https://jetpack.com/newsroom/',
```

Comment:
I don't see us using this consistently in Calypso with other Jetpack links, but what do you think about wrapping this in a [localizeUrl](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L201) function?

Reply:
Thanks for caring about localizing links. The `ExternalLink` component already handles that [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/external-link/index.jsx#L51).

Reply:
I was not aware of this, thank you for education me 😄